src/tests/ftest/pool/verify_space.yaml

```diff
@@ -0,0 +1,44 @@
+hosts:
+  test_servers: 3
+  test_clients: 1
+timeout: 600
+server_config:
+  # Test will require code changes if more than one rank/engine per host
+  engines_per_host: 1
+  engines:
+    0:
+      storage: auto
+  system_ram_reserved: 1
+pool_rank_0:
+  size: 100G
+  target_list: [0]
+  description: 'single pool on a single server (rank 0)'
+pool_rank_1_a:
+  size: 75G
+  target_list: [1]
+  description: 'multiple pools on a single server (rank 1)'
+pool_rank_1_b:
+  size: 85G
+  target_list: [1]
+  description: 'multiple pools on a single server (rank 1)'
```

Comment:
Looking at just the config, what is "a" vs "b"?

Reply:
Just different sized pools all targeting rank 1